RNTester/RNTester/RNTesterBundle/RNTesterTestPlan_iOS.xctestplan

```diff
@@ -0,0 +1,76 @@
+{
+  "configurations" : [
+    {
+      "id" : "8505E0BC-7E4D-4320-A0AB-AA632872F9B6",
+      "name" : "Default",
+      "options" : {
+
+      }
+    },
+    {
+      "id" : "37D31EEF-8FEF-4078-89E7-6BA48A640995",
+      "name" : "Memory Checking",
+      "options" : {
+        "addressSanitizer" : {
+          "detectStackUseAfterReturn" : true,
+          "enabled" : true
+        },
+        "nsZombieEnabled" : true
+      }
+    },
+    {
+      "id" : "B79DCEC6-8538-44AF-B29F-2F45036E8EE2",
+      "name" : "Concurrency",
+      "options" : {
+        "testExecutionOrdering" : "random",
+        "undefinedBehaviorSanitizerEnabled" : true
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "environmentVariableEntries" : [
+      {
+        "key" : "CI_USE_PACKAGER",
+        "value" : "1"
+      },
+      {
+        "key" : "RN_BUNDLE_PREFIX",
+        "value" : ""
+      }
+    ],
+    "targetForVariableExpansion" : {
+      "containerPath" : "container:RNTester.xcodeproj",
+      "identifier" : "13B07F861A680F5B00A75B9A",
+      "name" : "RNTester"
+    }
+  },
+  "testTargets" : [
+    {
+      "parallelizable" : true,
+      "skippedTests" : [
+        "RCTModuleInitTests\/testCustomInitModuleInitializedAtBridgeStartup",
+        "RCTModuleInitTests\/testLazyInitModuleNotInitializedDuringBridgeInit"
+      ],
+      "target" : {
+        "containerPath" : "container:RNTester.xcodeproj",
+        "identifier" : "004D289D1AAF61C70097A701",
+        "name" : "RNTesterUnitTests"
+      }
+    },
+    {
+      "parallelizable" : true,
+      "skippedTests" : [
+        "RNTesterIntegrationTests\/testAccessibilityManagerTest",
+        "RNTesterIntegrationTests\/testAppEventsTest",
+        "RNTesterIntegrationTests\/testSyncMethodTest",
+        "RNTesterIntegrationTests\/testWebSocketTest"
```

Comment:
Do you have a sense of the problem? Maybe it's easy to just fix?

Reply:
I'm missing from the diff/commit notes, but what exactly did we change so these could be re-enabled?

Reply:
Pushed them up while they were still failing. Tom pushed a header fix today that stopped the inconsistent failures